ports/stm32/adc.c

```diff
@@ -173,10 +173,13 @@
     defined(STM32H7B3xx) || defined(STM32H7B3xxQ) || \
     defined(STM32H750xx)
 #define VBAT_DIV (4)
-#elif defined(STM32L432xx) || \
+#elif defined(STM32L431xx) || \
+    defined(STM32L432xx) || defined(STM32L433xx) || \
+    defined(STM32L442xx) || defined(STM32L443xx) || \
     defined(STM32L451xx) || defined(STM32L452xx) || \
-    defined(STM32L462xx) || defined(STM32L475xx) || \
-    defined(STM32L476xx) || defined(STM32L496xx) || \
+    defined(STM32L462xx) || defined(STM32L471xx) || \
+    defined(STM32L475xx) || defined(STM32L476xx) || \
+    defined(STM32L486xx) || defined(STM32L496xx) || \
     defined(STM32L4A6xx) || \
```

Comment:
There reference manual confirms this is true for all STM32L4s, so suggest collapsing it to be `defined(STM32L4)`

![image](https://github.com/user-attachments/assets/a4261241-5e4e-4c1d-8db4-256aba7c1014)

Reply:
That's what I did in https://github.com/micropython/micropython/pull/7123